packages/expo-screen-capture/src/ScreenCapture.ts

```diff
@@ -0,0 +1,48 @@
+import { useEffect } from 'react';
+
+import ExpoScreenCapture from './ExpoScreenCapture';
+
+/**
+ * Prevents screen shots and screen recordings.
+ * (On iOS, this will only prevent screen recordings)
```

Comment:
Add a note that if you have already called this, calling it again does nothing. Also add a note that this method only works on iOS 11 and newer and doesn't do anything on older versions of iOS.